package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@domoinc/ryuu-proxy",
-  "version": "3.0.2",
+  "version": "3.0.3",
```

Comment:
Any reason we aren't making this a feature release?

Reply:
I see it kind of like a bug fix. The proxy should be able to handle requests. It was failing on multipart requests, and I fixed it. If you see it as more as a new feature I could change it!

Reply:
No, I think that's a reasonable explanation!